requirements.txt

```diff
@@ -27,16 +27,11 @@ seaborn>=0.11.0
 # scikit-learn==0.19.2  # CoreML quantization
 # tensorflow>=2.4.1  # TFLite export
 # tensorflowjs>=3.9.0  # TF.js export
+# openvino-dev  # OpenVINO export
 
 # Extras --------------------------------------
 # albumentations>=1.0.3
 # Cython  # for pycocotools https://github.com/cocodataset/cocoapi/issues/172
 # pycocotools>=2.0  # COCO mAP
 # roboflow
 thop  # FLOPs computation
-# CLI
```

Comment:
these dependencies are required and shouldnt be removed

Reply:
ok, I'll add it back (it was difficult for me to know what had been specifically added for the pip package -- and should be kept -- vs code that would have been removed from upstream! so there might be a  few other similar mistakes)